packages/cli/openapi-ir-sdk/fern/definition/finalIr.yml

```diff
@@ -175,10 +175,18 @@ types:
         docs: |
           Expected error status codes for this endpoint, and their corresponding schema and examples.
           SDK generators will only read the StatusCodes. Docs generators will read the HttpError schema.
-      server: list<commons.Server>
```

Comment:
Do we need to remove `commons.Server` or anything?

Reply:
It does look like the types are pretty similar, any reason for the new type? and why `url` is optional?

Reply:
Yup so `url` is now optional b/c the user can just do `x-fern-server-name: auth` on an endpoint instead of having to supply the URL whereas before we always mandated that they specify the URL since we were adhering to OpenAPI servers.